addons/mrp/models/mrp_production.py

```diff
@@ -1109,8 +1108,9 @@ def action_confirm(self):
             production.move_raw_ids._adjust_procure_method()
             (production.move_raw_ids | production.move_finished_ids)._action_confirm()
             production.workorder_ids._action_confirm()
-            # run scheduler for moves forecasted to not have enough in stock
-            production.move_raw_ids._trigger_scheduler()
+        # run scheduler for moves forecasted to not have enough in stock
+        self.move_raw_ids._trigger_scheduler()
```

Comment:
You could put only this line to avoid a big diff and risk no?

Reply:
Totally agree, I've updated the PR